hack/templates/values/values.template

```diff
@@ -52,9 +53,6 @@ node-feature-discovery:
       sources:
         pci:
           deviceClassWhitelist:
-          - "02"
-          - "0200"
-          - "0207"
```

Comment:
I would leave this, in case the users decide to set deployNodeFeatureRules to false. NFD can handle the duplicated config giving NodeFeatureRule priority

Reply:
I suggested to waleed to remove to avoid having duplicate places that label the same thing.

if this is left here, then why create nodefeaturerule by default. i think we want to rely on one mechanism for most cases.

since we may have external NFD deployed (which we require to have nodefeaturerules enabled) which then will require us to use nodefeaturerules created. i prefer to always rely on it.


if user decides to set this one to false, he needs to be aware of the implications. IMO.
in that case the user either: 
1. has another NFD deployed with all needed configs
2. user would need to change this entry to add the values needed 
3. or simply keep deployNodeFeatureRules true

Reply:
ACK, Done